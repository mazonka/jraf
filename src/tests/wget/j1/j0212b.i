command=jw md 0 /aaa/ : put 0 /aaa/bbb * 10 MTIzNDU2Nzg5MA== : put 0 /aaa/bbb * 10 MTIzNDU2Nzg5MA== : read 0 /aaa/bbb