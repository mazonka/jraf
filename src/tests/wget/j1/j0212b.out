OK /aaa : OK 10 : OK 20 : OK 2 20 MTIzNDU2Nzg5MDEyMzQ1Njc4OTA=
==
bbb
